app/views/unattended/provisioning_templates/provision/atomic_kickstart_default.erb

```diff
@@ -47,6 +47,7 @@ reboot
 rm -f /etc/ostree/remotes.d/*.conf
 <%= snippet('remote_execution_ssh_keys') %>
 <%= snippet 'efibootmgr_netboot' %>
+<%= snippet 'insights' -%>
```

Comment:
I'm not sure but I think insights is not packaged for atomic and the snippet wouldn't work. We should probably remove it.